### PR TITLE
Migrate terra badge

### LINKED ISSUE #14: Migrate terra-badge into repo

### PR DESCRIPTION
**Pull request by Matt-Butler**
Migrate badge to monorepo

This PR resolves #14 

@cerner/terra

### CODE REVIEW

**Comment by Matt-Butler** [comment]
https://github.com/cerner/terra-ui/issues/14

File: packages/terra-badge/lib/Badge.scss [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+.terra-Badge {
+  border-radius: $terra-badge-border-radius;
+  display: $terra-badge-display;
+  font-size: map-get($terra-badge-font-sizes, small);
+  font-weight: $terra-badge-font-weight;
+  line-height: $terra-badge-line-height;
+  min-width: 1.65em;
+  padding-bottom: $terra-badge-padding-bottom;
+  padding-left: $terra-badge-padding-right;
```
Comment (bjankord) [inline packages/terra-badge/lib/Badge.scss]: Looks like this needs to be switched over to use the bidi mixin:
`@include terra-padding-start($terra-badge-padding-left);`

File: packages/terra-badge/lib/Badge.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  padding-right: $terra-badge-padding-left;
```
Comment (bjankord) [inline packages/terra-badge/lib/Badge.scss]: Looks like this needs to be switched over to use the bidi mixin:
`@include terra-padding-end($terra-badge-padding-right);`

Comment (Matt-Butler) [inline packages/terra-badge/lib/Badge.scss]: https://github.com/cerner/terra-ui/pull/35/commits/e38d19c400ff0f1b1b3a7bd88c88a3109244e234

File: packages/terra-badge/lib/Badge.scss (hunk 1 above)
Comment (Matt-Butler) [inline packages/terra-badge/lib/Badge.scss]: https://github.com/cerner/terra-ui/pull/35/commits/e38d19c400ff0f1b1b3a7bd88c88a3109244e234

File: packages/terra-badge/README.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- Install with [npm](https://www.npmjs.com): `npm install terra-badge`
+
+## Documentation
+
+Documentation for this component is spilt into individual files.
+One file for the main component and one file for each component modifier.
+
+See the [documentation](docs/).
+
+## Supported Browsers
```
Comment (bjankord) [inline packages/terra-badge/README.md]: Lets remove this section as its in the root project README.md

File: packages/terra-badge/README.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+| Browser                     | Version |
+|-----------------------------|---------|
+| Chrome & Chrome for Android | Current |
+| Edge                        | Current |
+| Firefox                     | Current |
+| Internet Explorer           | 10 & 11 |
+| Safari & Mobile Safari      | Current |
+
+## Contributing
```
Comment (bjankord) [inline packages/terra-badge/README.md]: Lets remove this section as its in the root project README.md

**Pull request merged**